tooling/nargo/src/artifacts/debug.rs

```diff
@@ -45,6 +45,53 @@ impl DebugArtifact {
 
         Self { debug_symbols, file_map }
     }
+
+    /// Given a location, returns its file's source code
+    pub fn location_source_code(&self, location: Location) -> Result<&str, Error> {
+        Files::source(self, location.file)
```

Comment:
We don't tend to use the `Trait::method(self,...)` syntax, could you change this to `self.method(...)` instead please?